src/main/java/com/amplitude/HttpTransport.java

```diff
@@ -105,11 +145,10 @@ private CompletableFuture<Response> sendEvents(List<Event> events) {
           Response response = null;
           try {
             response = httpCall.makeRequest(events);
-            logger.debug("SEND", events, response);
+            logger.debug("SEND", "Events count " + events.size());
+            logger.debug("RESPONSE", response.toString());
           } catch (AmplitudeInvalidAPIKeyException e) {
             throw new CompletionException(e);
-          } catch (Exception e) {
```

Comment:
Just to double check, any reason that we want to remove this safe catch?

Reply:
moved to outer layer that call sendEvents() so I can handle unexpected exceptions in one place.